config/locales/en.yml

```diff
@@ -1139,3 +1135,15 @@ en:
           text: Advice on supporting children’s mental health during coronavirus (Young
             Minds)
           href: https://youngminds.org.uk/find-help/for-parents/supporting-your-child-during-the-coronavirus-pandemic/
+  feedback:
+    link_text: Give feedback on this service
+    link_href: https://www.gov.uk/done/find-coronavirus-support
+    titles:
+      feeling_unsafe: Was this information on what to do if you’re feeling unsafe or worried about someone else helpful?
```

Comment:
I wonder if there's a way to do this more programmatically. Not a blocker for this by any means but given that these are so generic / tied to the title of the page we might be able to cut down on this slightly at some point

Reply:
I did consider doing some sort of programmatic text construction as they all currently have the structure `"Was this information on #{page_specific_content} helpful?"`. However, I thought it very likely that the text would change and the cleverer the solution, the more difficult it would be to change the text structure. So in the end I went for the simple solution.

Reply:
Yeah, that makes sense, I think I would have ended up doing what you have here. Cards on the table I've always really disliked this file because of how long it is. Probably just me getting a bee in my bonnet

Reply:
Oh! I was very tempted to split this feedback segment out into it's own translation file. It just seemed out of keeping with the rest of the way the translations are presented. Breaking the file down into it three or four main segments would make a lot of sense to me, but I think that should be done in a separate PR. It might even be nice to have a translation file for each question - but that would require a big refactor.